Packs/HarfangLabEDR/Integrations/Hurukai/Hurukai.py

```diff
@@ -0,0 +1,2284 @@
+        'harfanglab-result-artifact-evtx': result_artifact_evtx,
+
+        'harfanglab-job-artifact-logs': job_artifact_logs,
+        'harfanglab-result-artifact-logs': result_artifact_logs,
+
+        'harfanglab-job-artifact-filesystem': job_artifact_fs,
+        'harfanglab-result-artifact-filesystem': result_artifact_fs,
+
+        'harfanglab-job-artifact-all': job_artifact_all,
+        'harfanglab-result-artifact-all': result_artifact_all,
+
+        'harfanglab-job-artifact-downloadfile': job_artifact_downloadfile,
+        'harfanglab-result-artifact-downloadfile': result_artifact_downloadfile,
+
+        'harfanglab-job-artifact-ramdump': job_artifact_ramdump,
+        'harfanglab-result-artifact-ramdump': result_artifact_ramdump,
+
+        'harfanglab-telemetry-processes': TelemetryProcesses().telemetry,
+        'harfanglab-telemetry-network': TelemetryNetwork().telemetry,
+        'harfanglab-telemetry-eventlog': TelemetryEventLog().telemetry,
+        'harfanglab-telemetry-binary': TelemetryBinary().telemetry,
+
+        'harfanglab-hunt-search-hash': hunt_search_hash,
+        'harfanglab-hunt-search-running-process-hash': hunt_search_running_process_hash,
+        'harfanglab-hunt-search-runned-process-hash': hunt_search_runned_process_hash,
+
+        'harfanglab-isolate-endpoint': isolate_endpoint,
+        'harfanglab-deisolate-endpoint': deisolate_endpoint,
+
+        'harfanglab-change-security-event-status': change_security_event_status,
+
+        'harfanglab-assign-policy-to-agent': assign_policy_to_agent,
+        'harfanglab-add-ioc-to-source': add_ioc_to_source,
+        'harfanglab-delete-ioc-from-source': delete_ioc_from_source,
+
+        'fetch-incidents': fetch_incidents,
+        'test-module': test_module
+    }
+
+    return commands.get(command)
+
+
+def main():
+    verify = not demisto.params().get('insecure', False)
+    proxy = demisto.params().get('proxy', False)
+    base_url = demisto.params().get('url').rstrip('/')
+    api_key = demisto.params().get('apikey')
+
+    try:
+        headers = {
+            'Authorization': f'Token {api_key}'
+        }
+
+        client = Client(
+            base_url=base_url,
+            verify=verify,
+            proxy=proxy,
+            headers=headers,
+        )
+
+        command = demisto.command()
+        target_function = get_function_from_command_name(command)
+
+        if target_function is None:
+            raise Exception('unknown command : {}'.format(command))
+
+        args = demisto.args()
+        if command == 'fetch-incidents':
+            args['first_fetch'] = demisto.params().get('first_fetch', None)
+            args['alert_status'] = demisto.params().get('alert_status', None)
+            args['alert_type'] = demisto.params().get('alert_type', None)
+            args['min_severity'] = demisto.params().get('min_severity', SEVERITIES[0])
+            args['max_results'] = demisto.params().get('max_results', None)
```

Comment:
Please rename to `max_fetch` to meet our standards.

Reply:
Done.